popup.js

```diff
@@ -0,0 +1,294 @@
+
+const projectCard1 = document.getElementById('projectCard1');
+const projectCard2 = document.getElementById('projectCard2');
+const projectCard3 = document.getElementById('projectCard3');
+const projectCard4 = document.getElementById('projectCard4');
+const projectCard5 = document.getElementById('projectCard5');
+const projectCard6 = document.getElementById('projectCard6');
+
+const projectCard = [
+  projectCard1,
+  projectCard2,
+  projectCard3,
+  projectCard4,
+  projectCard5,
+  projectCard6,
+];
+
+for (let i = 0; i < projectCard.length; i += 1) {
+  const cardSection = projectCard[i];
+
+  const card = document.createElement('div');
+  card.className = 'card';
+  card.id = 'project__grid';
+
+  const projectContainer = document.createElement('div');
+  projectContainer.className = 'project__container';
+  projectContainer.id = 'project_1';
+
+  const contentContainer = document.createElement('div');
+  contentContainer.className = 'content__container';
+
+  const projectContent = document.createElement('div');
+  projectContent.className = 'project__content';
+
+  const h3 = document.createElement('h3');
+  h3.innerHTML = data[2].cardTitle;
+
+  const p = document.createElement('p');
+  p.innerHTML = data[2].cardDescription;
+
+  const professionProject = document.createElement('div');
+  professionProject.className = 'profession__project-butons';
+
+  const cardBtn1 = document.createElement('button');
+  cardBtn1.innerHTML = data[2].cardBtn.cardBtn1;
+
+  const cardBtn2 = document.createElement('button');
+  cardBtn2.innerHTML = data[2].cardBtn.cardBtn2;
+
+  const cardBtn3 = document.createElement('button');
+  cardBtn3.innerHTML = data[2].cardBtn.cardBtn3;
+
+  const cardProjectBtn = document.createElement('button');
+  cardProjectBtn.className = 'see__project2';
+  cardProjectBtn.id = 'see__project2';
+  cardProjectBtn.innerHTML = data[2].cardProjectBtn;
+
+  cardSection.appendChild(card);
+  card.appendChild(projectContainer);
+  projectContainer.appendChild(contentContainer);
+  contentContainer.appendChild(projectContent);
+  projectContent.appendChild(h3);
+  projectContent.appendChild(p);
+  projectContent.appendChild(professionProject);
+  professionProject.appendChild(cardBtn1);
+  professionProject.appendChild(cardBtn2);
+  professionProject.appendChild(cardBtn3);
+  projectContainer.appendChild(cardProjectBtn);
+}
+
+// popup window
+
+const openPopup1 = document.getElementById('see__project1');
+const openPopup2 = document.getElementById('see__project2');
+
+const openPopups = [
+  openPopup1,
+  openPopup2,
+];
+
+for (let popup = 0; popup < openPopups.length; popup += 1) {
+  openPopups[popup].addEventListener('click', () => {
+    const mainDiv = document.createElement('div');
+    mainDiv.className = 'section';
+    const popup = document.createElement('div');
+    popup.className = 'popup__section';
+
+    const topContainer = document.createElement('div');
+    topContainer.className = 'topContainer';
```

Comment:
- [ ] As suggested by the previous reviewers, i would kindly suggest you stick to the project requirement as well as the Figma design as much as possible, Currently your project is not functioning the way it should function.

  - Only 2 out of the 7 `see project` buttons are functioning i.e display the popup card when clicked on, while the remaining 5 are not functioning at all.
  - Kindly make sure your popup cards align well to the screen and be sure it fit to the center of the screen as suggested by the previous reviewer and stated in the Figma design.
  - I would kindly suggest you work on the `mobile view` of the popup cards as it is not aligning properly to the mobile screen size of `375px`

Below is the video illustration of how your project looks like 👇 

[screen-capture (5).webm](https://user-images.githubusercontent.com/78634841/216482208-891687b0-2c74-4210-ab41-757fa9058c75.webm)